roles/drain_nodes/tasks/main.yml

```diff
@@ -3,6 +3,9 @@
   command: >-
     kubectl drain
     --ignore-daemonsets
+    {% if delete_emptydir_data|bool %}
```

Comment:
that's the same as delete_local_data but option has been renamed in kubectl. We need to link the two options together. Will think to change that